docs/docs/user/getting-started/components.md

```diff
@@ -0,0 +1,529 @@
+          repositoryName: bakdata-streams-bootstrap # required
+          url: https://bakdata.github.io/streams-bootstrap/ # required
+          repo_auth_flags:
+            username: user
+            password: pass
+            ca_file: /home/user/path/to/ca-file
+            insecure_skip_tls_verify: false
+        version: "2.7.0"
+    ```
+
+
+### deploy
+
+A Kubernetes job or cron job is deployed.
+The output topics are created, and schemas are registered in the Schema registry if configured.
+
+### destory
+
+The associated Kubernetes resources are removed.
+
+### reset
+
+Producers are not affected by reset as they are stateless.
+
+### clean
+
+The output topics of the Kafka producer are deleted as well as all associated schemas in the Schema Registry.
+
+## KafkaSinkConnector
+
+### Usage
+
+Lets other systems pull data from Apache Kafka.
+
+### Configuration
+
+??? "`pipeline.yaml`"
+
+    ```yaml
+      - type: kafka-sink-connector
+        name: kafka-sink-connector # required
+        namespace: namespace # required
+        # `app` contains application-specific settings, hence it does not have a rigid
+        # structure. The fields below are just an example.
+        app: # required
+          key1:
+            key4: value3
+          key2: value1
+          key3: value2
+        # Topic(s) from which the component will read input
+        from:
+          topics: # required
+            ${pipeline_name}-input-topic:
+              type: input # required
+              role: topic-role
+        # Topic(s) into which the component will write output
+        to:
+          topics: # required
+            ${pipeline_name}-output-topic:
+              type: output # required
+              role: topic-role
+              keySchema: key-schema
+              valueSchema: value-schema
+              partitions_count: 1
+              replication_factor: 1
+              configs: 
+                config1: config1
+          models: 
+            model: model
+        # Pipeline prefix that will prefix every component name. If you wish to not 
+        # have any prefix you can specify an empty string.
+        prefix: ${pipeline_name}-
+        # Helm repository configuration
+        repoConfig:
+          repositoryName: my-repo # required
+          url: https://bakdata.github.io/kafka-connect-resetter/ # required
+          repoAuthFlags: 
+            username: user
+            password: pass
+            ca_file: /home/user/path/to/ca-file
+            insecure_skip_tls_verify: false
+        version: "1.0.4"
+        # Overriding Kafka Connect Resetter Helm values. E.g. to override the
+        # Image Tag etc.
+        resetterValues:
+          imageTag: 1.2.3
+        offsetTopic: offset_topic
+    ```
+
+### deploy
+
+A Kafka connector is deployed.
+The output topics are created, and schemas are registered in the Schema registry if configured.
+
+### destory
+
+The associated Kafka connector is removed.
```

Comment:
This can move to KafkaConnector

Reply:
As agreed, there wouldn't be such a section. I guess in that case this should be kept here?